libraries/AP_Common/Location.h

```diff
@@ -108,6 +108,20 @@ class Location
     // return true when lat and lng are within range
     bool check_latlng() const;
 
+    // see if location is past a line parallel to
+    // the line between point1 and point2. If point1 is
+    // our previous waypoint and point2 is our target waypoint
+    // then this function returns true if we have flown past
```

Comment:
s/flown/travelled/

Reply:
// see if location is past a line perpendicular to
// the line between point1 and point2 and passing through point2.
// If point1 is
// our previous waypoint and point2 is our target waypoint
// then this function returns true if we have flown past
// the target waypoint